docs/dm/zh_CN/features/table-selector.md

```diff
@@ -0,0 +1,29 @@
+table selector
+===
+
+#### 功能介绍
+
+table selector 对 schema/table 提供了一种基于通配符(https://zh.wikipedia.org/wiki/%E9%80%9A%E9%85%8D%E7%AC%A6)的匹配规则
```

Comment:
```suggestion
table selector 对 schema/table 提供了一种基于通配符(https://zh.wikipedia.org/wiki/%E9%80%9A%E9%85%8D%E7%AC%A6) 的匹配规则
```

or 

```suggestion
table selector 对 schema/table 提供了一种基于 [通配符](https://zh.wikipedia.org/wiki/%E9%80%9A%E9%85%8D%E7%AC%A6)的匹配规则
```

now, the like is `https://zh.wikipedia.org/wiki/%E9%80%9A%E9%85%8D%E7%AC%A6)%E7%9A%84%E5%8C%B9%E9%85%8D%E8%A7%84%E5%88%99`